projects/packages/videopress/src/client/block-editor/blocks/video/edit.native.tsx

```diff
@@ -46,7 +55,13 @@ export default function VideoPressEdit( {
 	);
 
 	if ( isUploadingFile ) {
-		return <VideoPressUploader handleDoneUpload={ handleDoneUpload } />;
+		return (
+			<VideoPressUploader
+				autoOpenMediaUpload={ isSelected && wasBlockJustInserted }
```

Comment:
This condition matches what we have in other media blocks ([reference](https://github.com/WordPress/gutenberg/blob/584c1d27a92db85491fe4bf3eb883d689faa3f5f/packages/block-library/src/video/edit.native.js#L247-L249)).